<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BenchSortingWindow</class>
 <widget class="QWidget" name="BenchSortingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="benchAllButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>830</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Bench all</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 28px;
font-weight: 500;
text-decoration: underline;
color: #e66739;</string>
   </property>
   <property name="text">
    <string>Counting sort</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>830</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>‚Üê Go back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 28px;
font-weight: 500;
text-decoration: underline;
color: #e66739;</string>
   </property>
   <property name="text">
    <string>Comparison sort</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="comparisonTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>440</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #DE6262,  stop:1 #FFB88C);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QTableView {
    selection-background-color: #dedede;

	selection-color: dark-grey;

	font-size:15px;
}


/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}

QTableView::item {
}
</string>
   </property>
   <column>
    <property name="text">
     <string>N</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sort by</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="radixTable">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>180</y>
     <width>440</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #DE6262,  stop:1 #FFB88C);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QTableView {
    selection-background-color: #dedede;

	selection-color: dark-grey;

	font-size:15px;
}


/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}

QTableView::item {
}
</string>
   </property>
   <column>
    <property name="text">
     <string>N</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sort by</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="pageName_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1501</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: orange;
font-size: 35px;
font-family: Roboto;</string>
   </property>
   <property name="text">
    <string>Sorting benchmark</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="countingTable">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>440</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #DE6262,  stop:1 #FFB88C);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QTableView {
    selection-background-color: #dedede;

	selection-color: dark-grey;

	font-size:15px;
}


/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}

QTableView::item {
}
</string>
   </property>
   <column>
    <property name="text">
     <string>N</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sort by</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>110</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 28px;
font-weight: 500;
text-decoration: underline;
color: #e66739;</string>
   </property>
   <property name="text">
    <string>Radix sort</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>830</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	background-color: #e1e1e1;
border-radius: 5px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="numberRadio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="typeRadio">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>25</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="numberTypeRadio">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>25</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number &gt; Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
font-weight: 500;</string>
    </property>
    <property name="text">
     <string>Sort by</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="typeNumberRadio">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>25</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Type &gt; Number</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>830</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	background-color: #e1e1e1;
border-radius: 5px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
font-weight: 500;</string>
    </property>
    <property name="text">
     <string>Array size</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sortingArrSizeCounter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>100000000</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
